README.md

```diff
@@ -1,93 +1,24 @@
-# `@ubiquibot/plugin-template`
+# `@ubiquibot/automated-merging`
 
-## Prerequisites
+Automatically merge pull-requests based on the reviewer count, the time elapsed since the last activity, depending 
+on the association of the pull-request author.
 
-- A good understanding of how the [kernel](https://github.com/ubiquity/ubiquibot-kernel) works and how to interact with it.
-- A basic understanding of the Ubiquibot configuration and how to define your plugin's settings.
-
-## Getting Started
-
-1. Create a new repository using this template.
-2. Clone the repository to your local machine.
-3. Install the dependencies preferably using `yarn` or `bun`.
-
-## Creating a new plugin
-
-- If your plugin is to be used as a slash command which should have faster response times as opposed to longer running GitHub action tasks, you should use the `worker` type.
-
-1. Ensure you understand and have setup the [kernel](https://github.com/ubiquity/ubiquibot-kernel).
-2. Update [compute.yml](./.github/workflows/compute.yml) with your plugin's name and update the `id`.
-3. Update [context.ts](./src/types/context.ts) with the events that your plugin will fire on.
-4. Update [plugin-inputs.ts](./src/types/plugin-inputs.ts) to match the `with:` settings in your org or repo level configuration.
-
-- Your plugin config should look similar to this:
+## Configuration example
 
 ```yml
-- plugin: <plugin-org/owner>/<plugin-repo-name>:compute.yml@development
-  name: plugin-name
-  id: plugin-name-command
-  description: "Plugin description" # small description of what the plugin does
-  command: "<regex for command>" # if you are creating a plugin with a slash command
-  example: "<example usage>" # how to invoke the slash command
-  with: # these are the example settings, the kernel passes these to the plugin.
-    disabledCommands: []
-    timers:
-      reviewDelayTolerance: 86000
-      taskStaleTimeoutDuration: 2580000
-    miscellaneous:
-      maxConcurrentTasks: 3
-    labels:
-      time: []
-      priority: []
+- plugin: ubiquibot/automated-merging
+  name: automated-merging
+  id: automated-merging
+  description: "Automatically merge pull-requests"
+  with:
+    collaboratorMinimumApprovalsRequired: 1
+    contributorMinimumApprovalsRequired: 2
+    collaboratorMergeTimeout: "3.5 days"
+    contributorMergeTimeout: "7 days"
```

Comment:
```suggestion
  with:
    approvalsRequired:
        collaborator: 1
        contributor: 2
    mergeTimeout:
        collaborator: "3.5 days"
        contributor: "7 days"
```

This seems more clear to me. Thoughts?

Reply:
@0x4007 looks better imo, will update.

Reply:
Updated at https://github.com/ubiquibot/automated-merging/pull/1/commits/0592aee06b5d920795da60f7c989065a7aa58ca4